app/src/components/InputElement/InputElement.styles.js

```diff
@@ -3,12 +3,35 @@ import background from "../../images/bg-shorten-desktop.svg";
 
 export const StyledForm = styled.form`
   position: absolute;
-  left: 270px;
+  left: 128px;
+  top:705px;
```

Comment:
formatting is not correct. Sometimes, there is a space between property and value and sometimes not

Reply:
already corrected with prittier.